tests/methods/avas-6/input.dat

```diff
@@ -1,7 +1,7 @@
 #! Test AO Subspace regular expression on 3d orbitals
 
 refscf = -1646.626167484472944
-reffci = -1646.647217416001240
+reffci = -1646.647075290021348
```

Comment:
This is suspicious. Why would the energy go up if you make the active space larger? Is FCI locking onto an excited state? Or am I missing something?

Reply:
Good catch. I've also noticed this.

I think there are two things related. One is the AO subspace did not recognize C10 before so that the projector is not the same as before.

The second thing is related to the AVAS bug fix in PR #219, where an important orbital is not chosen into the active space by AVAS. Combining with the semicanonicalization after AVAS, this will change the CASCI energy (albeit the HF energy is invariant and this invariance is verified). Now which CASCI energy is lower? I am not sure because the orbitals are different.

Reply:
Ok, thank you for the explanation.